### ISSUE TITLE
```GET /records``` var-{variable ID}-{value ID} filter is not working

### ISSUE THREAD

**Issue by ghost** [speedruncom/api#153]
I'm trying to make a world record leaderboard for every game but the request link doesn't seem to work, as the runs aren't filtered by the variable specified.
Please someone take a look at this link:
https://www.speedrun.com/api/v1/games/k6q4rqzd/records?skip-empty=true&max=200&offset=0&top=1&embed=players&var-2lggye3l=4lxexp31
![image](https://user-images.githubusercontent.com/102447359/176247036-16e10c4b-7bf2-43a7-82ce-c282929efae7.png)

**Comment by Nick-NCSU** [comment]
Assuming you are referring to `GET /games/{id}/records` which it seems your request uses: refer to the query parameters listed for this endpoint (https://github.com/speedruncomorg/api/blob/master/version1/games.md#get-gamesidrecords). The only documented query parameters are `top` `scope` `miscellaneous` `skip-empty` `max` `offset` and `embed`.

The `var-___` query parameter is only available for the `GET /leaderboards/{game}/category/{category}` and `GET /leaderboards/{game}/level/{level}/{category}` endpoints.

**Comment by danthedrummer** [comment]
Those query params aren't documented in the `/games` route. Should you be using the `/leaderboards` route instead? 

Something like this:
https://www.speedrun.com/api/v1/leaderboards/k6q4rqzd/category/z27lez0d?skip-empty=true&max=200&offset=0&top=1&embed=players&var-2lggye3l=4lxexp31

_edit: Nick beat me to it_ 😂

**Comment by ghost** [comment]
Is there any way to filter all top 1 runs by variables then?

I though every propriety of the lbs doc also applied here lmao

**Comment by Nick-NCSU** [comment]
No, you would need to iterate through each one using the `GET /leaderboards/{game}/category/{category}` endpoint with `top=1`. The `GET /games/{id}/records` returns the fastest run in a _category_ not subcategory.

**Comment by ghost** [comment]
Oh well that's not good for me, also doesn't really make sense to not return the fastest run for every subcategory that has each lb
